bucket/snaptimer.json

```diff
@@ -0,0 +1,23 @@
+{
+    "version": "0.1",
+    "description": "SnapTimer - Free Windows Countdown Timer",
+    "homepage": "https://dan.hersam.com/software/snaptimer/index.php",
+    "license": "Freeware",
+    "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip",
+    "hash": "227df1414eefda1d98249d6e96465266a9b6517e29808e0eb57f3eca8875d23b",
+    "extract_dir": "SnapTimer",
+    "bin": "SnapTimer.exe",
+    "shortcuts": [
+        [
+            "SnapTimer.exe",
+            "SnapTimer"
+        ]
+    ],
+    "checkver": {
+        "url": "https://dan.hersam.com/software/snaptimer/index.php",
+        "regex": "<b>Version:</b> (\\d+\\.\\d+)"
+    },
+    "autoupdate": {
+        "url": "https://cdn.hersam.com/software/snaptimer/SnapTimer.zip"
```

Comment:
Investigate version specific URL

Reply:
couldn't find it, I just follow some other JSON files in the `bucket` directory. :|